pennylane/templates/subroutines/qpe.py

```diff
@@ -200,6 +201,34 @@ def estimation_wires(self):
         """The estimation wires of the QPE"""
         return self._hyperparameters["estimation_wires"]
 
+    def error(self):
+        """The QPE error computed from the spectral norm error of the input unitary operator.
```

Comment:
We can come back to the documentation and add some more of the details when we advertise the feature